Taskfile.yaml

```
@@ -0,0 +1,192 @@
+  migrate-clean:
+    desc: "Clean migration directory"
+    cmds:
+      - rm -rf migrations
+      - mkdir -p migrations
+
+  migrate-init:
+    desc: "Initialize migration files"
+    cmds:
+      - task: gorm-schema
+      - |
+        atlas migrate diff initial_schema --env development 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)